specs/util.spec.js

```diff
@@ -76,5 +76,42 @@ describe('util', () => {
     it('should thrown an error if the file does not exists', () => {
       expect(() => util.loadFile('./there-no-this-file')).to.throw('Could not load the file: "./there-no-this-file". One of the next files must exists: ./there-no-this-file,./there-no-this-file.js,./there-no-this-file.yaml,./there-no-this-file.yml');
     });
+  });
+
+  describe('FakerClass', () => {
```

Comment:
What happens when the `!faker` invocation is bad? An exception will fire, but will it assist a dev in fixing the mistake? If the default exception contains the bad value, the it is good enough. If it does not (and just says something bad happened), then the framework should add the original invocation.

Reply:
I have added a test for that if an error happens, we should see an error like ```Error parsing the file ./fixtures/fixture-with-faker-invalid.data.yaml: cannot resolve a node with !<!faker> explicit tag at line 3, column 1:\n    \n```.  Let me know if you want me to change this validation. For now, I am using the default `js-yaml` message.   Please check the tets https://github.com/testbetter/openapitest/pull/12/commits/4bfdf827011d648396f4c99e88f5007d2fc76455#diff-72600850c21863fc1c33a850cda580a6R47 and the code change in https://github.com/testbetter/openapitest/pull/12/commits/4bfdf827011d648396f4c99e88f5007d2fc76455#diff-020982b33423dafe61ea5851c63acc46R247